test_resources/schemas/module-name.txt

```diff
@@ -0,0 +1 @@
+- [33m<string>[m
```

Comment:
I thought `toComponents` would produce a list, now I'm a bit confused.

Reply:
mhm you are right. it appears that both `String` and `[String]` are represented as `<string>`. is this something i should worry about?

Reply:
Ah no! There's a `-` in front, which means list. That makes sense then.
It was just harder to see with the ansi codes.